CRM/Core/BAO/Setting.php

```diff
@@ -1033,8 +1055,9 @@ protected static function getOverride($group, $name, $default) {
     global $civicrm_setting;
     if ($group && $name && isset($civicrm_setting[$group][$name])) {
       return $civicrm_setting[$group][$name];
-    }
-    else {
+    } else if ($group && !isset($name) && isset($civicrm_setting[$group])) {
```

Comment:
quick note on this - check our whitespace standard for spacing around 'else' - our standards are drupal so ...
https://www.drupal.org/coding-standards